mailman/submit.py

```diff
@@ -39,9 +39,32 @@ def main(argv):
 
     # if the 'cover letter' is not addressed to
     # an assignment inbox, this email session
-    # isn't a patchset at all
+    # isn't a patchset at all, but it could be a peer review.
     if cover_letter.rcpt not in ASSIGNMENT_LIST:
-        # TODO process peer review
+        if patches:
+            return 0
+        with open(f'/var/lib/email/mail/{cover_letter.msg_id}', 'r') as f:
+            for line in f:
+                if line == '':
+                    return 0
+                if line.startswith('In-Reply-To: '):
```

Comment:
maybe `if not line.startswith('In-Reply-To: '): continue` so the rest can be unindented?